pkg/management/component.go

```diff
@@ -552,21 +566,14 @@ func (c *component) GetUser(ctx context.Context, realmName, userID string) (api.
 	userRep.IDDocumentNumber = dbUser.IDDocumentNumber
 	userRep.IDDocumentExpiration = dbUser.IDDocumentExpiration
 	userRep.IDDocumentCountry = dbUser.IDDocumentCountry
-	userRep.PendingChecks = keycloakb.ConvertFromDBChecks(pendingChecks).ToCheckNames()
+	userRep.PendingChecks = keycloakb.ConvertFromAccreditationChecks(pendingChecks).ToCheckNames()
 
 	//store the API call into the DB
 	c.reportEvent(ctx, "GET_DETAILS", database.CtEventRealmName, realmName, database.CtEventUserID, userID, database.CtEventUsername, username)
 
 	return userRep, nil
 }
 
-func (c *component) isUpdated(newValue, oldValue *string) bool {
-	if oldValue == nil {
-		return newValue != nil
-	}
-	return newValue != nil && *oldValue != *newValue
-}
-
 func (c *component) UpdateUser(ctx context.Context, realmName, userID string, user api.UpdatableUserRepresentation) error {
```

Comment:
I have the feeling this method could now be simplified, to discuss

Reply:
Hum not trivial to do, we can maybe split it in small submethods